skopt/tree_opt.py

```diff
@@ -207,6 +230,19 @@ def forest_minimize(func, dimensions, base_estimator='rf', maxiter=100,
     * `n_points` [int, default=1000]:
         Number of points to sample when minimizing the acquisition function.
 
+    * `x0` [list, shape=(n_initial,)]:
+        List of initial input parameters used to bootstrap the surrogate
+        model. Note that if `x0` is provided, no random initialization
+        points are used, so in that case `n_start` is ignored.
```

Comment:
Oh, should we ignore `n_start`? Sometimes the user might want to use bootstrap along with random initialization. For this use case, we should make the user explicitly supply `n_start=0` instead of implicitly ignoring a parameter.